components/input/theme.css

```diff
@@ -0,0 +1,280 @@
+
+/**
+ * Variables
+ */
+:root {
+  --input-height-large: calc(4.6 * var(--unit));
+  --input-height-medium: calc(3.4 * var(--unit));
+  --input-height-small: calc(2.8 * var(--unit));
+  --input-width: calc(30 * var(--unit));
+  --input-horizontal-padding: calc(0.9 * var(--unit));
+  --input-horizontal-padding-small: calc(0.6 * var(--unit));
+  --input-horizontal-padding-icon: calc(3.9 * var(--unit));
+  --input-horizontal-padding-icon-small: calc(2.6 * var(--unit));
+  --input-text-color: var(--color-teal-darkest);
+  --input-text-size: calc(1.6 * var(--unit));
+  --input-text-size-small: calc(1.4 * var(--unit));
+  --input-background: var(--color-white);
+  --input-border-radius: calc(0.4 * var(--unit));
+  --input-border-color: var(--color-neutral-dark);
+  --input-placeholder-color: var(--color-neutral-dark);
+  --input-focus-border: var(--color-aqua);
+  --input-active-border: var(--color-neutral-darkest);
+  --input-active-box-shadow: rgba(130, 130, 140, 0.5);
+  --input-error-background: var(--color-ruby-lightest);
+  --input-error-border: var(--color-ruby-dark);
+
+  --input-disabled-background: var(--color-neutral);
+  --input-disabled-border: var(--color-neutral-dark);
+  --input-disabled-color: var(--color-neutral-darkest);
+  --input-read-only-background: var(--color-neutral);
+  --input-read-only-border: var(--color-neutral-dark);
+
+  --icon-top-large: calc(1.4 * var(--unit));
+  --icon-top: calc(0.7 * var(--unit));
+  --icon-horizontal: calc(0.9 * var(--unit));
+  --icon-horizontal-small: calc(0.6 * var(--unit));
+  --icon-height: calc(1.9 * var(--unit));
+  --icon-height-small: calc(1.4 * var(--unit));
+  --icon-color: var(--color-neutral-darkest);
+
+  --counter-margin-top: calc(0.7 * var(--unit));
+  --counter-margin-small: calc(0.5 * var(--unit));
+  --counter-margin-large: calc(1.4 * var(--unit));
+  --counter-margin-left: calc(0.5 * var(--unit));
+  --counter-width: calc(3 * var(--unit));
+
+  --spinner-width: calc(2.9 * var(--unit));
+  --spinner-width-large: calc(3.6 * var(--unit));
+  --spinner-background: var(--color-neutral-light);
+  --spinner-border: var(--color-neutral-dark);
+  --spinner-color: var(--color-neutral-darkest);
+
+  --error-font-size: calc(1.4 * var(--unit));
+  --error-color: var(--color-ruby-dark);
+}
+
+/**
+ * Components
+ */
+.wrapper {
+  position: relative;
+
+  width: 100%;
+  max-width: var(--input-width);
+}
+
+.input {
+  transition: 0.3s ease-in-out border, 0.3s ease-in-out box-shadow;
+
+  position: relative;
+  box-sizing: border-box;
+
+  height: var(--input-height-medium);
+  width: 100%;
+  padding: 0 var(--input-horizontal-padding);
+
+  background: var(--input-background);
+
+  border: 1px solid var(--input-border-color);
+  border-radius: var(--input-border-radius);
+  box-shadow: 0;
+
+  color: var(--input-text-color);
+  line-height: var(--input-height-medium);
+  font-size: var(--input-text-size);
+  font-family: var(--font-family-regular);
+
+  &:focus {
```

Comment:
you also need to disable the outline here for Chrome.

Reply:
👍🏻

Reply:
Done.